### ISSUE TITLE
GQ Server/Client 0.10.0 Pre 4 - Stations missing in client display

### ISSUE THREAD

**Issue by andyaz** [xspanger3770/GlobalQuake#169]
I have notices that all of the stations that are green and selected in the station selection screen do not appear on the client screen. They appear in local mode but not client/server.

Attached is the server select screenshot and a client display screenshot. You can easily see the missing stations.

Select creen on Server:
![station select](https://github.com/xspanger3770/GlobalQuake/assets/19541773/baa86533-0284-4105-b9aa-e5c901fb1920)

Client display of the same general area:
![Mising Stations](https://github.com/xspanger3770/GlobalQuake/assets/19541773/c1415555-1ed7-4df7-9438-3cf84275290f)

**Comment by xspanger3770** [comment]
So after a bit of investigation, I figured out that the problem is that IRIS seedlink (or maybe all seedlinks) have a limit on how many stations can be selected (around 1000) in one connection, and since the stations load in lexicographic order based on network name, the networks with highest names (UO, UU, TW) wont load at all.

**Comment by xspanger3770** [comment]
Solved in pre15: You can now add multiple instances of the same seedlink and the stations will automatically get balanced between the connections

**Comment by andyaz** [comment]
I disagree. Even with multiple IRIS seedlink entries it has the same behavior from what I see.
![image](https://github.com/xspanger3770/GlobalQuake/assets/19541773/cc592bb3-5593-430c-8376-aefdadea8fbe)

**Comment by andyaz** [comment]
I will give it a try when I get home later.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by andyaz** [comment]
Thanks for that tidbit. It is working now. Thanks sir!
